client/src/includes/initIconSprite.ts

```diff
@@ -0,0 +1,45 @@
+export const initIconSprite = (
+  spriteContainer: HTMLElement,
+  spriteURL: string,
+) => {
+  const revisionKey = 'wagtail:spriteRevision';
+  const dataKey = 'wagtail:spriteData';
+  const isLocalStorage: boolean =
```

Comment:
I know this came from other code, but I suggest we rename this to `hasLocalStorage` as it reads better.

```suggestion
  const hasLocalStorage: boolean =
```